hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerReplicaInfo.java

```diff
@@ -45,7 +46,9 @@ public static ContainerReplicaInfo fromProto(
         .setPlaceOfBirth(UUID.fromString(proto.getPlaceOfBirth()))
         .setSequenceId(proto.getSequenceID())
         .setKeyCount(proto.getKeyCount())
-        .setBytesUsed(proto.getBytesUsed());
+        .setBytesUsed(proto.getBytesUsed())
+        .setReplicaIndex(
```

Comment:
I think we can just set 0 also, as that indicate non EC containers? EC containers replicaIndex will start from 1

Reply:
I was using "-1" as a case to indicate that the index is not in the proto at all. Ie, SCM is at an older version than the client. Ideally this should not happen except during an upgrade window from pre-EC to the EC release, and then never again.

When displaying the index, I have some logic to not display it at all if its -1. Otherwise display it even if it is zero.

Reply:
It make sense. +1